app/src/main/java/com/getdefault/androidtasks/FragmentA.kt

```diff
@@ -0,0 +1,89 @@
+package com.getdefault.androidtasks
+
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.getdefault.androidtasks.databinding.FragmentABinding
+
+class FragmentA : Fragment() {
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        val binding = FragmentABinding.inflate(layoutInflater)
+
+        Log.d("Fragment_A", "OnCreateView Called")
```

Comment:
@saurav-kalsoor-default instead of writing Fragment_A again and again, you should create a constant and use it. Decreases chances of typos drastically.

Reply:
Ok Sir.